### PR TITLE
feat: nginx ingress controller quickstart

### PR DESCRIPTION
**Pull request by tclayton-newr**
# Summary

Adding a quickstart for NGINX ingress controller metrics gathered using Prometheus. 

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [ ] ~Did you check that your alerts actually work?~
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

See quickstarts/kubernetes/nginx/dashboards/nginx-ingress-dashboard.png

### CODE REVIEW

File: quickstarts/kubernetes/nginx/config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: NGINX Ingress Controller
```
Comment (aswanson-nr) [inline quickstarts/kubernetes/nginx/config.yml:1]: This just needs to called `slug` and be hyphenated lowercase
```suggestion
slug: nginx-ingress-controller
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @tclayton-newr ! 
This looks almost good to go, just noticed one small needed change in the config.yml

File: quickstarts/kubernetes/nginx/config.yml (hunk 1 above)
Comment (tclayton-newr) [reply quickstarts/kubernetes/nginx/config.yml:1]: Oops, I'll get that fixed!

File: quickstarts/kubernetes/nginx/config.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+documentation:
+  - name: NGINX Ingress Controller Quickstart
+    url: https://kubernetes.github.io/ingress-nginx/deploy/#quick-start
+    description: Details on installing the NGINX ingress controller in your Kubernetes cluster
+  - name: Enabling metrics for your NGINX ingress controller
+    url: https://kubernetes.github.io/ingress-nginx/user-guide/monitoring/#prometheus-and-grafana-installation-using-pod-annotations
+    description: Details on enabling metrics for your NGINX ingress controller 
+
+# Content / Design
+logo: logo.svg
```
Comment (aswanson-nr) [inline quickstarts/kubernetes/nginx/config.yml:50]:
```suggestion
icon: logo.svg
```

**Pull request merged**